30413/HulpeMiruna/5.3/main.c

```diff
@@ -0,0 +1,209 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+const int MAXNAME=200;
+
+typedef struct
+{
+    char *name;
+    float price;
+    int amount;
+    int dateOfManufacturing;
+    int dateOfExpiration;
+} Pharmacy;
```

Comment:
this actually represents a `Drug` or `Medicine` not the whole `Pharmacy`.. I would've used `Pharmacy` instead of `NodeT` below